all-collections/_our-team/meg-gutshall.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+
+first_name: Meg
+last_name: Gutshall
+list_order: H
+position: Open Source Advocate and Welcome Wagoner
+photo: /assets/img/people/meg-gutshall.png
+email: meg@rubyforgood.org
+social:
```

Comment:
For the time being we're not including social links for RfG people until we're able to start coordinating their social media presences.